composer.json

```diff
@@ -30,9 +30,9 @@
         "league/pipeline": "^0.3.0",
         "monolog/monolog": "~1.6",
         "padraic/phar-updater": "^1.0",
-        "phpdocumentor/flyfinder": "@beta",
-        "phpdocumentor/graphviz": "~1.0",
-        "phpdocumentor/reflection": "dev-develop@dev",
+        "phpdocumentor/flyfinder": "1.0.0-beta2",
```

Comment:
you want to suffix this with `@beta` So you can use `minimum-stabiltity: stable` this prevents us from using other packages from any alpha or betha phase.

Reply:
These explicit alphas are temporary, just testing stage stepping stones.  Once phpdoc2 itself proves out, I'll do beta release tags, set them to beta in here, and ultimately we can have them all as final releases.